channeldb/channel.go

```diff
@@ -1223,7 +1223,9 @@ func (c *OpenChannel) RemoteCommitChainTip() (*CommitDiff, error) {
 	err := c.Db.View(func(tx *bolt.Tx) error {
 		chanBucket, err := readChanBucket(tx, c.IdentityPub,
 			&c.FundingOutpoint, c.ChainHash)
-		if err != nil {
+		if err == ErrNoActiveChannels || err == ErrNoChanDBExists {
```

Comment:
Not sure how we could encounter these if we're manipulating the object of an active channels...but I guess it can't hurt

Reply:
It can happen during some of the unit tests.